test/unit/model/RdsResultStorageTest.php

```diff
@@ -271,24 +271,16 @@ public function testStoreItemVariable()
         $variables = $this->instance->getVariable($callId, $identifier);
 
         $object = array_shift($variables);
-        $this->assertEquals($test, $object->test);
-        $this->assertEquals($item, $object->item);
-        $this->assertInstanceOf(OutcomeVariable::class, $object->variable);
-        $this->assertEquals($baseType, $object->variable->getBaseType());
-        $this->assertEquals($cardinality, $object->variable->getCardinality());
-        $this->assertEquals($identifier, $object->variable->getIdentifier());
-        $this->assertEquals($value, $object->variable->getValue());
+        $this->assertItemVariableBasics($test, $object, $item);
+
+        $this->assertVariable($baseType, $object, $cardinality, $identifier, $value);
+
 
-        $this->assertEquals($baseType, $this->instance->getVariableProperty($object->uri, 'baseType'));
-        $this->assertEquals($cardinality, $this->instance->getVariableProperty($object->uri, 'cardinality'));
-        $this->assertEquals($identifier, $this->instance->getVariableProperty($object->uri, 'identifier'));
-        $this->assertEquals($value, $this->instance->getVariableProperty($object->uri, 'value'));
-        $this->assertNull($this->instance->getVariableProperty($object->uri, 'unknownProperty'));
     }
 
     public function testStoreItemVariableException()
     {
-        $this->expectException(DuplicateVariableException::class);
+        $this->setExpectedException(DuplicateVariableException::class);
```

Comment:
I think you should update generis aand start using PHPUnit 8.5

Reply:
Indeed; I was on the latest `generis`, but with old `phpunit`;